src/xmipp/bindings/python/python_image.cpp

```diff
@@ -1894,6 +1896,50 @@ Image_applyGeo(PyObject *obj, PyObject *args, PyObject *kwargs)
 }
 
 
+PyObject *
+Image_window2D(PyObject *obj, PyObject *args, PyObject *kwargs)
+{
+    ImageObject *self = (ImageObject*) obj;
+    if (nullptr == self) return nullptr;
+    try {
+        // keep default values consistent with the python
+        int x0 = 0;
+        int y0 = 0;
+        int xF = 384;
+        int yF = 384;
```

Comment:
Giving these values by default seems to be unnatural. I would make all of them compulsory, instead of optional.